style.css

```diff
@@ -0,0 +1,299 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: "poppins";
+}
+ 
+   :root {
+    --dark-beige:  #E6DACD;
+    --light-beige: #F4ECE6;
+    --royal-blue: #0150FD;
```

Comment:
Good work using CSS vars, might make the names descriptive of the use rather than the color so you can change the theme with one small change without confusing people but that's a minor gripe